pvlib/iam.py

```diff
@@ -748,3 +748,108 @@ def marion_integrate(function, surface_tilt, region, num=None):
         Fd = pd.Series(Fd, surface_tilt.index)
 
     return Fd
+
+
+def fedis(aoi, surface_tilt, n=1.5, n_ref=1.4585):
+    """
+    Determine the incidence angle modifiers (iam) for direct, diffuse sky,
+    and ground-reflected radiation using the FEDIS transmittance model.
+
+    The "Fresnel Equations" for Diffuse radiation on Inclined photovoltaic
+    Surfaces (FEDIS) [1]_ is an analytical solution of diffuse transmission
+    based on the rigorous integration of an alternate form of the
+    Fresnel equations. The approach leads to a simple yet accurate
+    relative transmittance model that reconciles the solar energy
+    sensed by pyranometers and PV panels.
+
+    Parameters
+    ----------
+    aoi : numeric
+        Angle of incidence. [degrees]
+
+    surface_tilt : numeric
+        Surface tilt angle measured from horizontal (e.g. surface facing
+        up = 0, surface facing horizon = 90). [degrees]
+
+    n : float, default 1.5
+        Refractive index of the PV cover.  The default value of 1.5
+        was used for an IMT reference cell in [1]_. [unitless]
+
+    n_ref : float, default 1.4585
+        Refractive index of the pyranometer cover. The default value
+        was used for a fused silica dome over a CMP22 in [1]_.
+
+    Returns
+    -------
+    iam : dict
+        IAM values for each type of irradiance:
+
+            * 'direct': radiation from the solar disc
```

Comment:
Minor preference here for 'irradiance' instead of 'radiation', only for consistency of terms.